src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -158,7 +167,9 @@
      * towards the root.
      */
     private static final int[][] ALLOWED_ASSIGNMENT_IN_COMPARISON_CONTEXT = {
-        {TokenTypes.EXPR, TokenTypes.LITERAL_WHILE, },
+            {TokenTypes.EXPR, TokenTypes.LITERAL_WHILE},
```

Comment:
Can we put the indentation back since there was no real change here?

Reply:
fixed